src/lib_ccx/ccx_decoders_common.c

```diff
@@ -202,9 +204,7 @@ int do_cb(struct lib_cc_decode *ctx, unsigned char *cc_block, struct cc_subtitle
 				{
 					if (ctx->write_format != CCX_OF_RCWT)
 					{
-#ifdef ENABLE_RUST
-						dtvcc_process_cc_data(ctx->dtvcc, (const unsigned char *)temp);
-#else
+#ifndef ENABLE_RUST
 						dtvcc_process_data(ctx->dtvcc, (const unsigned char *)temp);
```

Comment:
Why do we have two separate functions? dtvcc_process_cc_data is the rust version now?

Reply:
I realize that the naming for rust function might be confusing. There's only a single rust function called inside C code, named dtvcc_process_cc_data